### DISCUSSION TITLE
onedrive不能用了

### DISCUSSION THREAD

**Discussion by sail5200919** [AlistGo/alist#1354] [Q&A] [answered]
今晚已经弄过的onedrive显示错误，我也看不懂，就索性删了重新弄
但是现在文档里提供的获取刷新令牌的网站，一直在获取中

**Comment by sail5200919** [comment, marked as answer]
已解决

**Reply by cckunkun** [reply]
朋友，一直获取中解决办法是什么呢？

**Reply by sail5200919** [reply]
> 朋友，一直获取中解决办法是什么呢？

很早的了，自己也忘了，不过我刚刚用我的弄过的去试了一下，是可以获取的
确定数值都对的话，可能是他这个服务有点慢，搜搜别的能获取onedrive刷新令牌的，我记得之前搜过到一些别的

**Reply by cckunkun** [reply]
谢谢，应该是找到原因了，我刚才试了试有可能是缓存的原因，重新开了无痕窗口就可以了。几个月前用过登录的另一个账号，等于获取令牌的时候应该是默认连了之前的账号
